src/pages/overview/pdf-services-api/howtos/index.md

```diff
@@ -172,9 +172,6 @@ Available properties:
     milliseconds for creating an initial HTTPS connection.
 -   **socketTimeout**: Default: 10000. The maximum allowed time in
     milliseconds between two successive HTTP response packets.
--   **processingTimeout**: Default: 600000. The maximum allowed time
-    in milliseconds for processing the documents. Any operation taking more time than the specified `processingTimeout` will result in an operation timeout exception.
```

Comment:
why was this removed ? we can see processing timeout in config and the code snippet also not chnaged accordingly ?

Reply:
As discussed with Anubhav this is removed from new Java SDK